src/main/java/Event.java

```diff
@@ -1,18 +1,23 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Event extends Task {
 
-    private String time;
+    private final LocalDateTime time;
 
-    public Event(String command, String time) {
+    public Event(String command, LocalDateTime time) {
         super(command);
```

Comment:
There is no documentation in your code, maybe try to take a look at the java documentation to give others a clear idea on the functionality of the classes. 👊